zip-0227.rst

```diff
@@ -85,7 +85,9 @@ Issuance Authorization Signature Scheme
 
 The requirement of cryptographic authorization in order to issue new Assets or change attributes of existing Assets is met by using a signature to prove knowledge of the issuance key used to authorize the issuance of the Asset, similar to what is done with the Spend Authorization Signature [#protocol-spendauthsig]_.
 
-We define the issuance authorization signature scheme :math:`\mathsf{IssueAuthSig}` identically to the Orchard spend authorization signature scheme, :math:`\mathsf{SpendAuthSig^{\mathsf{Orchard}}`, that is, as :math:`RedPallas` with key re-randomization and with generator :math:`\mathcal{P}_{\mathbb{G}} = \mathcal{G}^{\mathsf{Orchard}}` [#protocol-concretespendauthsig]_.
+We define the issuance authorization signature scheme :math:`\mathsf{IssueAuthSig}` similar to the Orchard spend authorization signature scheme, :math:`\mathsf{SpendAuthSig}^{\mathsf{Orchard}}` [#protocol-concretespendauthsig]_. 
+Specifically, we instantiate :math:`\mathsf{IssueAuthSig}` as :math:`\mathsf{RedPallas}` with key re-randomization and with generator :math:`\mathcal{P}_{\mathbb{G}} = \mathcal{G}^{\mathsf{Issuance}} := \mathsf{GroupHash}^\mathbb{P}(\texttt{"z.cash:ZSA"}, \texttt{"Issuance"})` where :math:`\mathsf{GroupHash}^\mathbb{P}` is defined as in [#protocol-concretegrouphashpallasandvesta]_.
```

Comment:
On instantiating the IssueAuthSig as RedPallas with key re-randomization, are issuance keys ever re-randomized?

Reply:
Edited based on our discussion to mention it is without key re-randomization.